### ISSUE TITLE
Technique G74, Title and Examples (SC 1.1.1)

### ISSUE THREAD

**Issue by toddmarquisboutin** [w3c/wcag#2129]
I created this issue on behalf of the [Alt Text Subgroup](https://www.w3.org/WAI/GL/task-forces/silver/wiki/Alt_Text_Subgroup) of the Silver Task Force.
### Explanation:
The title for [Technique G74](https://www.w3.org/WAI/WCAG21/Techniques/general/G74.html) currently reads as follows: “Providing a long description in text near the non-text content, with a reference to the location of the long description in the short description.”

In other WCAG techniques, long descriptions are treated as separately identifiable elements. Refer to Examples 1 and 2 of [Technique G73](https://www.w3.org/TR/WCAG20-TECHS/G73.html). Example 1 recommends placing the long description in a separate section of the page titled “Descriptions of charts on this page.” Example 2  recommends placing the long description on a different, dedicated page titled “Description of charts…”

As a result, upon reading the term “long description” in the title of G74, some people might assume that the extended figure description must be separate from the body text. This is not the case, however. The relevant details of the non-text content can also be provided in the image caption and body text. 

Often, using the body text to describe the image can improve the user experience by making that information more readily available in context (i.e., by directly integrating the figure information into the text it complements). Also, by providing the details of non-text content in the body text, content authors can reduce the text-alternative requirement for a figure.
### Recommendation:
Change the title of Technique G74 to “Providing the details of the non-text content in the text near it, while explaining the location of the details in the alt text.”

This more general title covers both methods: (a) provide the long description in a separate section on the page, (b) include the figure details in the image caption and/or body text.

We also recommend adding an example that demonstrates how the relevant figure details can be provided in the image caption and body text (method b).

**Comment by GreggVan** [comment]
Question

Long descriptions are fine when using HTML  — but how do you do them in Powerpoint and Excel and many other forms that are showing up on the internet now. 

And WCAG is used for document access too so all document formats are fair game.   We got  ALT text into them  but not long desc

Is the advice to just put long descriptions into ALT text if there is no known (to the author) way to attach a long desc to the image?

Thx

gregg

———————————
Professor, University of Maryland, College Park
Founder and Director Emeritus , Trace R&D Center, UMDCo-Founder Raising the Floor. http://raisingthefloor.org
The Global Public Inclusive Infrastructure (GPII) http://GPII.net
The Morphic project  https://morphic.org

**Comment by toddmarquisboutin** [comment]
@GreggVan, thanks for your question.

When you mention "long desc," are you specifically referring to the longdesc attribute?

Long descriptions can be delivered in a variety of ways. In this GitHub issue, we refer to a method that works for both HTML and non-HTML products: [WCAG Technique G74](https://www.w3.org/WAI/GL/2013/WD-WCAG20-TECHS-20130711/G74).

G74 suggests the following approach:
1. Place the long description on the page (or slide or sheet) near the image.
2. Use the "short" alt text to (a) summarize the contents or purpose of the image and (b) tell the user where to find the long description.

**Example**: A complex diagram appears on a PPT slide. The author places the long description for the diagram in the Speaker Notes for the slide on which the diagram appears. The author then writes "short" alt text that summarizes the purpose of the diagram. The alt text ends by stating, "Refer to the Speaker Notes for a detailed description of the diagram."

In Issue #2129, we are suggesting a slight change to the wording of the Technique title (and the addition of an example) to clarify the fact that a detailed description for a complex figure can often be seamlessly integrated into the body text and/or figure caption, as opposed to being set apart on the page.

[WCAG Technique G73](https://www.w3.org/WAI/WCAG21/Techniques/general/G73) provides another approach that works in both HTML and non-HTML formats. In this case, the long description is placed in another location, and the user accesses the long description via a link immediately adjacent to the image.

**Example**: A complex graph in a PDF contains a large amount of vital data. An internal link immediately follows the image in the reading order of the page. The link text reads "Data table for the graph." The link leads to an appendix page that contains a properly tagged table presenting the graph data. The table is accompanied by an introductory statement that explains how the structure and contents of the table relate to the graph.

Techniques G73 and G74 have the added advantage of making the long description available to people who listen to the content **and** people who view the content.

**Comment by mraccess77** [comment]
Longer descriptions in my opinion would ideally be put into the content visually when something is created.   These can be exposed through disclosure and other techniques.  They can also be linked, embedded, or provided and referenced in the shorter description.

Longer descriptions are often put into the notes of slides.

Sometimes it is not possible and this information may need to be presented in a non-visual way.  If the format doesn't have multiple ways to communicate identity/purpose and equivalent alternatives then it may need to be provided in the single way that the format does support.

The format of content needs to support these different mechanisms and the viewers of the format need to support presenting this information as well.

**Comment by GreggVan** [comment]
Great idea to use the Notes section in slides.   I used to do that and forgot about it.   So have been putting long descriptions in ALT text. 
The Alt text could also include a note at the end saying "Longer description in notes section of slide"  

Still doesn't solve the problem of needing a longer description for diagrams where the text won’t allow it — (editor won’t allow or there is a character count or page limit that would be broken….)  

Would be nice if we can get EVERY format that allows ALT text field to ALSO allow a long description field….  

gregg

———————————
Professor, University of Maryland, College Park
Founder and Director Emeritus , Trace R&D Center, UMDCo-Founder Raising the Floor. http://raisingthefloor.org
The Global Public Inclusive Infrastructure (GPII) http://GPII.net
The Morphic project  https://morphic.org
